projector-plugin/build.gradle.kts

```diff
@@ -44,3 +45,32 @@ intellij {
 (tasks["runIde"] as JavaExec).apply {
   jvmArgs = jvmArgs.orEmpty() + listOf("-Djdk.attach.allowAttachSelf=true", "-Dswing.bufferPerWindow=false")
 }
+
+abstract class GenerateVersionsFile: DefaultTask() {
+  private val filePath = "src/main/resources/META-INF/versions.txt"
```

Comment:
I suggest making this file name specific to the module (e.g. `pluginVersions.txt`). This will help to avoid situations when files with the same name are overwritten, for example, when making fat jars.

Reply:
Fixed